apps/acrs/index.js

```diff
@@ -668,7 +668,6 @@ module.exports = {
       backLink: false
     },
     '/session-expired': {
-      fields: [],
```

Comment:
I think we can remove the '/session-expired' step from here altogether, as the form currently renders it as an error on the form page that you were last on.

Reply:
done